html_app/ui/ExperimentSetupView.js

```diff
@@ -297,7 +297,7 @@ scb.ui.static.ExperimentSetupView.scb_f_open_experiment_setup_readonly = functio
 			else{
 				$('body').prepend(scb_experiment_setup.experiment_setup_overlay());
 				$(element).attr('href', 'javascript:void(0)');
-				
+				$('#jqDialog_box').css('width', '570px');
```

Comment:
I'm not crazy about hard-coding CSS, especially when it depends on a font rendering.